dis_snek/utils/misc_utils.py

```diff
@@ -74,3 +74,10 @@ def wrap_partial(obj, cls):
         obj.subcommands = {k: wrap_partial(v, cls) for k, v in obj.subcommands.items()}
 
     return obj
+
+
+def get_parameters(callback: Callable):
```

Comment:
Maybe dict comprehension would be better/faster

Reply:
I'd prefer to leave it in its own method ( i intend to reuse this), but ill convert it to use comprehension instead